src/HotspotEditor.tsx

```diff
@@ -446,9 +450,14 @@ function HotspotEditor({
       )}
 
       <HotspotDataEditor
-        hotspotData={previewData}
-        setHotspotData={updateData}
-      />
+  hotspotData={previewData}
+  setHotspotData={(data) => {
+    setPreviewData(data);
+    setEdited(true);
+  }}
```

Comment:
This is supposed to be `setHotspotData={updateData}`.

Reply:
Changed